init/onboard-central-ledger/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "airbnb-base",
```

Comment:
@elnyry-sam-k do we have somewhere specified lint standard for mojaloop projects?

Reply:
@eoln we have a `.eslintrc.js` config here: https://docs.mojaloop.io/documentation/contributors-guide/standards/#config-files, but not 100% sure if that is up-to-date.